public/app/features/dashboard/components/PanelEditor/FieldConfigEditor.tsx

```diff
@@ -178,13 +182,17 @@ export class FieldConfigEditor extends React.PureComponent<Props> {
   };
 
   render() {
+    const { plugin } = this.props;
+
     return (
       <div>
-        <OptionsGroup title="Field configuration">{this.renderStandardConfigs()}</OptionsGroup>
+        {plugin.customFieldConfigs && (
+          <OptionsGroup title={`${plugin.meta.name} options`}>{this.renderCustomConfigs()}</OptionsGroup>
+        )}
 
-        {this.props.custom && <OptionsGroup title="Visualization options">{this.renderCustomConfigs()}</OptionsGroup>}
+        <OptionsGroup title="Field defaults">{this.renderStandardConfigs()}</OptionsGroup>
```

Comment:
The words "defaults" and "overrides" are likely worth the effort to get some UX research/feedback on how to best explain this!

Reply:
yea,  I know. but since we are using overrides defaults makes at least some sense. Maybe "Field settings" and "Field overrides" are better. definitely something for UX to research